jobsdb/backup.go

```diff
@@ -256,7 +260,10 @@ func (jd *Handle) backupJobsTable(ctx context.Context, backupDSRange *dataSetRan
 	}()
 	err = jd.uploadDumps(ctx, dumps)
 	if err != nil {
-		jd.logger.Errorf("[JobsDB] :: Failed to upload dumps for table: %s. Error: %v", tableName, err)
+		jd.logger.Errorw(
+			"Failed to upload backup",
```

Comment:
Let's be consistent, all logs should either start with a lowecase or an uppercase character